stellarphot/photometry/tests/test_profiles.py

```diff
@@ -117,27 +158,82 @@ def test_radial_profile_exposure_is_nan():
 def test_radial_profile_with_background():
     # Regression test for #328 -- image with a background level
     image = make_gaussian_sources_image(SHAPE, STARS)
+    noise_stdev = 10
     image = image + make_noise_image(
-        image.shape, distribution="gaussian", mean=100, stddev=0.1
+        image.shape, distribution="gaussian", mean=100, stddev=noise_stdev, seed=43917
     )
     for row in STARS:
         cen = find_center(image, (row["x_mean"], row["y_mean"]), max_iters=10)
 
         # The "stars" have FWHM around 9.5, so make the cutouts used for finding the
         # stars fairly big -- the bare minimum would be a radius of 3 FWHM, which is a
         # cutout size around 60.
-        rad_prof = CenterAndProfile(image, cen, cutout_size=60, profile_radius=30)
+        rad_prof = CenterAndProfile(
+            image, cen, centering_cutout_size=60, profile_radius=30
+        )
 
         # Numerical value below is integral of input 2D gaussian, 2pi A sigma^2
         expected_integral = 2 * np.pi * row["amplitude"] * row["x_stddev"] ** 2
 
+        # The standard deviation in the sum of N gaussian random variables with
+        # standard deviation SD is
+        #    σ = sqrt(N × SD^2)
+        # The curve of growth includes the sume of a bunch of pixels which each have a
+        # standard deviation of 10, so the standard deviation of the sum of those pixels
+
+        expected_stddev = np.sqrt(rad_prof.curve_of_growth.area[-1] * noise_stdev**2)
+
+        # Allow for a 3-sigma tolerance
         np.testing.assert_allclose(
-            rad_prof.curve_of_growth.profile[-1], expected_integral, atol=50
+            rad_prof.curve_of_growth.profile[-1],
+            expected_integral,
+            atol=3 * expected_stddev,
         )
 
         # Test that the radial profile is correct by comparing pixel values to a
         # gaussian fit to the profile.
         data_radii, data_counts = rad_prof.pixel_values_in_profile
         expected_profile = rad_prof.radial_profile.gaussian_fit(data_radii)
 
-        np.testing.assert_allclose(data_counts, expected_profile, atol=20)
+        # The test here is that the difference between the actual profile and the
+        # expected is itself a Gaussian distribution with standard deviation very
+        # roughly equal to the standardof the noise we put in.
+        differences = data_counts - expected_profile
+        counts, bin_edges = np.histogram(differences)
+        bin_centers = (bin_edges[1:] + bin_edges[:-1]) / 2
+        g1d_init = Gaussian1D()
+        fitter = LevMarLSQFitter()
+        g1d = fitter(g1d_init, bin_centers, counts)
+
+        assert np.abs(g1d.stddev.value - noise_stdev) < 1.5
+        assert np.abs(g1d.mean.value) < 1
+
+
+def test_radial_profile_bigger_profile_than_cutout():
+    # Test that the cutout, used for finding the star, can be smaller than
```

Comment:
I am not following how this is a test that the cutoff can be smaller than the profile radius. 

From reviewing the code of this test, it looks like you are testing that if the cutout is smaller than needed for the profile radius, the size of the cutout will be adjusted accordingly.  Can you clarify the comment accordingly.